.gitignore

```diff
@@ -41,3 +41,5 @@ dist/
 .vscode
 
 .DS_Store
+.editorconfig
```

Comment:
Let's not .gitignore these globally. If they bother you, you can ignore locally (e.g.: `.git/info/exclude`). We'll add them to the repo soon.

Reply:
done